src/vumi2/routers.py

```diff
@@ -49,6 +50,15 @@ async def setup(self):
                 connector_name=name, outbound_handler=self.handle_outbound_message
             )
 
+        if (
+            self.config.default_app
+            and self.config.default_app not in self.receive_outbound_connectors
+        ):
```

Comment:
Would putting `self.config.default_app` in a local variable make this fit on one line?